src/components/landing-page/use-fetch-production-list.ts

```diff
@@ -21,8 +21,14 @@ export const useFetchProductionList = () => {
 
           setProductions(
             result
-              // pick laste 10 items
-              .slice(-10)
+
+              // Make sure list is correctly sorted
+              .sort(
+                (a, b) =>
+                  parseInt(b.productionId, 10) - parseInt(a.productionId, 10)
+              )
```

Comment:
this should not be needed, the manager API should return data in the order we expect

@birme did something change recently with the listing API? We made sure it would return the 50 most recently created productions in order

Reply:
The backend returns the data in correct order. That has been verified.

Reply:
```
curl -X 'GET' \
  'https://intercom-manager.dev.eyevinn.technology/api/v1/production' \
  -H 'accept: application/json'
```

returns
```
[
  {
    "name": "r",
    "productionId": "76"
  },
  {
    "name": "q",
    "productionId": "75"
  },
  {
    "name": "p",
    "productionId": "74"
  },
  {
    "name": "o",
    "productionId": "73"
  },
  {
    "name": "n",
    "productionId": "72"
  },
  {
    "name": "m",
    "productionId": "71"
  },
  ...
```

so there must be somewhere in the client where JSON-parsing does not maintain the order. What saying it should?

Reply:
ordered list (array) order should be maintained according to json spec, if it was delievered as a keyed object it would be another matter. Strange issue, I'll ask @malmen237 to show me what's going on.